### PR TITLE
Fix UI for release tag page / wiki page / subscription page

### PR DESCRIPTION
**Pull request by puni9869**
Agenda:
This PR contains UI fixes for release tag page / wiki page / subscription page.
Here is the list of changes made in this PR.
1. Release tag page
a. In the New Release page the whole ui got change. Now it is covering in full page page with mobile view port. Description about the release the editor preview now has a min-height. and the check boxes for `Prerelease`   and  option are left aligned.  Couple of divider are added.

2. Subscription page: 
a. In the subscription page the ui was distorted in mobile view. Now its fix. Couple of unused styles were removed.

3. Create Wiki page:- 
a. In the page the preview of markdown is now contains a fix min-height so this it will not distorted in desktop view and a divider  is added before action buttons. Couple of unused styles were removed.

# Before

## Release page

image

image

image

image

Editor 
image

Editor preview
image

image

#### After
image

image
image
image

image

Markdown preview change
image

Wiki page

Before
image
image

Preview of mark down. 
image

Footer 
image

After

image
image
image

Preview tab block has min-height 
image

Mobile view
image

image
image

### Before

image

image

After
image

image

image

### CODE REVIEW

**Comment by puni9869** [comment]
Note:- Please take the pull of this branch for better understanding and then go through the ui changes.

File: web_src/css/repo/release-tag.css [hunk 1]
```diff
@@ -132,3 +132,8 @@
 .repository.new.release .field .attachment_edit {
   width: 450px !important;
 }
+
+.repository.new.release .markup {
+  min-height: 240px;
+  overflow: visible;
```
Comment (silverwind) [inline web_src/css/repo/release-tag.css]: Why the need to override overflow? Is it coming from Fomantic?

Comment (puni9869) [reply web_src/css/repo/release-tag.css]: Sorry for the late response, its not required hence, Removed.

**Review by delvh — APPROVED** [review]

**Comment by puni9869** [comment]
Needs one more 🪄  approval

**Review by silverwind — APPROVED** [review]

**Pull request merged**